demo/list/bullet/playground

```diff
@@ -0,0 +1,28 @@
+const listItems = [
+  {
+    illustration: 'https://s.ccdn.es/images/motivation-search.svg',
+    title: 'Publica con total seguridad',
+    text:
+      'Te damos todas las herramientas para un proceso de venta seguro'
+  },
+  {
+    illustration: 'https://s.ccdn.es/images/motivation-trust.svg',
+    title: 'Gestiona tu anuncio según tus necesidades',
+    text: 'Edita y renueva tu anuncio tantas veces como necesites'
+  },
+  {
+    illustration: 'https://s.ccdn.es/images/motivation-security.svg',
+    title: 'Chatea con los compradores potenciales',
+    text:
+      'Resuelve rápidamente las dudas de los interesados en tu anuncio en el chat'
+  }
+]
+
+return (
+  <div>
+    <h2>Standard</h2>
+    <ListBullet listItems={listItems}/>
+    <h2>Responsive</h2>
+    <ListBullet listItems={listItems} responsive />
```

Comment:
as you can use every exported object from your src/index.js, you should we able to create something like

```jsx
<ListBullet>
  <Bullet illustration="https://s.ccdn.es/images/motivation-search.svg" title="Publica con total seguridad" text="Te damos todas las herramientas para un proceso de venta seguro" />
  <Bullet illustration="https://s.ccdn.es/images/motivation-trust.svg" title="Gestiona tu anuncio según tus necesidades" text="Edita y renueva tu anuncio tantas veces como necesites" />
  <Bullet illustration="https://s.ccdn.es/images/motivation-security.svg" title="Chatea con los compradores potenciales" text="Resuelve rápidamente las dudas de los interesados en tu anuncio en el chat" />
</ListBullet>
```

Reply:
As we spoke, i don't see the improvement decoupling a component logic that already has a coupled styles. i would see the improvement iterating isolated components.
anyways, thanks for taking time to explaining this pattern. i would use it for sure.